packages/fluent/scss/list/_variables.scss

```diff
@@ -0,0 +1,243 @@
+@use "../core/color-system" as *;
+@use "../core/_variables.scss" as *;
+
+/// Font size of the list component, if no size is set.
+/// @group list
+$kendo-list-font-size-sm: var( --kendo-font-size, inherit ) !default;
+$kendo-list-font-size-md: var( --kendo-font-size, inherit ) !default;
+$kendo-list-font-size-lg: var( --kendo-font-size, inherit ) !default;
+
+/// Line height of the list component, if no size is set.
+/// @group list
+$kendo-list-line-height-sm: var( --kendo-line-height, normal ) !default;
+$kendo-list-line-height-md: var( --kendo-line-height, normal ) !default;
+$kendo-list-line-height-lg: var( --kendo-line-height, normal ) !default;
+
+/// Horizontal padding of list header, if no size is set.
+/// @group list
+$kendo-list-header-padding-x-sm: map-get( $kendo-spacing, 2 ) !default;
+$kendo-list-header-padding-x-md: map-get( $kendo-spacing, 2 ) !default;
+$kendo-list-header-padding-x-lg: map-get( $kendo-spacing, 2 ) !default;
+
+/// Vertical padding of list header, if no size is set.
+/// @group list
+$kendo-list-header-padding-y-sm: map-get( $kendo-spacing, 1 ) - map-get( $kendo-spacing, thin ) !default;
+$kendo-list-header-padding-y-md: map-get( $kendo-spacing, 1 ) !default;
+$kendo-list-header-padding-y-lg: map-get( $kendo-spacing, 1 ) + map-get( $kendo-spacing, thin ) !default;
+
+/// Border width of list header.
+/// @group list
+$kendo-list-header-border-width: 0 0 1px !default;
+
+/// Font size of list header, if no size is set.
+/// @group list
+$kendo-list-header-font-size-sm: null !default;
+$kendo-list-header-font-size-md: null !default;
+$kendo-list-header-font-size-lg: null !default;
+
+/// Line height of list header, if no size is set.
+/// @group list
+$kendo-list-header-line-height-sm: null !default;
+$kendo-list-header-line-height-md: null !default;
+$kendo-list-header-line-height-lg: null !default;
+
+/// Font weight of list header.
+/// @group list
+$kendo-list-header-font-weight: $kendo-font-weight-bold !default;
+
+/// Horizontal padding of list items, when no size is set.
+/// @group list
+$kendo-list-item-padding-x-sm: map-get( $kendo-spacing, 2 ) !default;
+$kendo-list-item-padding-x-md: map-get( $kendo-spacing, 2 ) !default;
+$kendo-list-item-padding-x-lg: map-get( $kendo-spacing, 2 ) !default;
+
+/// Vertical padding of list items, when no size is set.
+/// @group list
+$kendo-list-item-padding-y-sm: map-get( $kendo-spacing, 1 ) - map-get( $kendo-spacing, thin ) !default;
+$kendo-list-item-padding-y-md: map-get( $kendo-spacing, 1 ) !default;
+$kendo-list-item-padding-y-lg: map-get( $kendo-spacing, 1 ) + map-get( $kendo-spacing, thin ) !default;
+
+/// Font size of list items, if no size is set.
+/// @group list
+$kendo-list-item-font-size-sm: null !default;
+$kendo-list-item-font-size-md: null !default;
+$kendo-list-item-font-size-lg: null !default;
+
+/// Line height of list items, if no size is set.
+/// @group list
+$kendo-list-item-line-height-sm: null !default;
```

Comment:
We should refrain from using null values  as fallbacks for css variables and instead use the default value for the given property.